ffi/diplomat/src/segmenter_line.rs

```diff
@@ -59,50 +59,153 @@ pub mod ffi {
     pub struct ICU4XLineBreakIteratorLatin1<'a>(LineBreakIteratorLatin1<'a, 'a>);
 
     impl ICU4XLineSegmenter {
```

Comment:
Praise: Thanks for updating the FFI and the FFI tests. They look great

Reply:
Thanks!